swagger-contract-tests/src/test/resources/contract/swagger/declaration-pet-service.json

```diff
@@ -304,7 +304,7 @@
                             "paramType": "query",
                             "format": "int32",
                             "type": "integer",
-                            "required": false
+                            "required": true
```

Comment:
A number of these tests had invalid contract specs, as a result of the prior incorrect behavior of required parameters.